test/restore.bats

```diff
@@ -100,7 +100,7 @@ function teardown() {
 	stop_crio
 
 	# simulate reboot with runc state going away
-	"$CONTAINER_RUNTIME" delete -f "$pod_id"
+	"$RUNTIME_BINARY_PATH" --root "$RUNTIME_ROOT" delete -f "$pod_id"
```

Comment:
so, since you're using `"$RUNTIME_BINARY_PATH" --root "$RUNTIME_ROOT"` from many places, makes sense to have something like

```bash
function runtime() {
   "$RUNTIME_BINARY_PATH" --root "$RUNTIME_ROOT" "$@"
}
```
and use it everywhere. For example, the above would become
```suggestion
	runtime delete -f "$pod_id"
```
which I think helps readability as well as maintenance (making future patches smaller).